recipes/interpolation/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: interpolation
+  version: "0.1.8"
+source:
+  git_url: https://github.com/econforge/interpolation.py
+  git_tag: v0.1.8
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  preserve_egg_dir: True
+  number: 1
+  skip: True  # [py>35]
+requirements:
+  build:
+    - python
+    - setuptools
+    - numba
+    - numpy
+    - pandas
+    - cython
+    - scipy
+    - tempita
+    - pip
+  run:
+    - python
+    - numba
+    - numpy
+    - pandas
+    - cython
```

Comment:
It appears as if `cython` is only used in `misc` which is not part of the distributed package.